charts/camunda-platform/values/values-v8.0.yaml

```diff
@@ -26,11 +26,6 @@ optimize:
   image:
     repository: camunda/optimize
     tag: 3.9.4
-  env:
-  # TODO: Remove the suffix when it's fixed in Optimize.
-  # https://github.com/camunda/camunda-platform-helm/issues/714
-  - name: CAMUNDA_OPTIMIZE_IDENTITY_REDIRECT_ROOT_URL
```

Comment:
Do we need to remove the whole var or just the endpoint `/api/authentication/callback`?

Reply:
probably just the suffix actually, have changed 👍